bin/index.js

```diff
@@ -132,7 +132,12 @@ function speedText(speed) {
 /*
  * Function that return state of Spinner, and change its state with time
  */
-var frames = ['⠁', '⠂', '⠄', '⡀', '⢀', '⠠', '⠐', '⠈' ];
+var frames = [
```

Comment:
These fancy dots don't work in Windows. Could you add a Windows version?
```
var frames = process.platform === 'win32' ? [
    '|',
    '/',
    '-',
    '\\'
  ] : [
    '⠂⠁⠂⠄',
    '⠄⠂⠁⠂',
    '⠂⠄⠂⠁',
    '⠁⠂⠄⠂',
  ];
```
Or if you think of something fancier that doesn't require Unicode, then that'll work for me as well.